src/ExpoVideoMetadata.types.ts

```diff
@@ -7,6 +7,11 @@ export type VideoInfoResult = {
    * Tells if the video has a audio track. If the video has no audio track, its considered a mute video.
    */
   hasAudio: boolean;
+  /**
+   * Available only on iOS >= 14 and Android. Tells if the video is a HDR video.
+   * Will return null if it could not be determined.
+   */
+  isHDR: boolean | null;
```

Comment:
I guess there is no support for web, right? I still would like to return null on web, to have a identical API.

https://github.com/hirbod/expo-video-metadata/blob/bf4d28c81e9703f65481f567c68bca04c130d8e1/src/ExpoVideoMetadataModule.web.ts#L195-L196

Reply:
Yes - you're right!

Reply:
... my latest commit should address this